modules/kubernetes-cluster/outputs.tf

```
@@ -37,3 +37,7 @@ output "node_resource_group" {
 output "location" {
   value = "${azurerm_kubernetes_cluster.main.location}"
 }
+
+output "http_application_routing_zone_name" {
+  value = "${azurerm_kubernetes_cluster.main.addon_profile.0.http_application_routing.0.http_application_routing_zone_name}"
```

Does this throw an error if there is no routing zone? We might want to add the same conditional as used in the creation (I haven't tested this though)
```suggestion
  value = var.enable_http_application_routing ? azurerm_kubernetes_cluster.main.addon_profile.0.http_application_routing.0.http_application_routing_zone_name : ""
```